dandiapi/api/tests/test_asset.py

```diff
@@ -35,130 +35,44 @@ def test_asset_blob_and_zarr(draft_asset, zarr_archive):
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize(
-    'path,asset_paths,expected',
-    [
-        ('', ['foo', 'bar/baz'], {'folders': ['bar'], 'files': ['foo']}),
-        # ('', ['/foo', '/bar/baz'], {'folders': ['bar'], 'files': ['foo']}),  # edge case
-        ('/', ['/foo', '/bar/baz'], {'folders': ['bar'], 'files': ['foo']}),
-        ('', ['foo/bar', 'foo/baz', 'foo/boo'], {'folders': ['foo'], 'files': []}),
-        ('/', ['foo/bar', 'foo/baz', 'foo/boo'], {'folders': [], 'files': []}),  # Negative test
-        ('////', ['/foo', '/bar/baz'], {'folders': [], 'files': []}),
-        ('a', ['a/b', 'a/c/d'], {'folders': ['c'], 'files': ['b']}),
-        ('a/', ['a/b', 'a/c/d'], {'folders': ['c'], 'files': ['b']}),
-        ('/a', ['/a/b', '/a/c/d'], {'folders': ['c'], 'files': ['b']}),
-        ('/a/', ['/a/b', '/a/c/d'], {'folders': ['c'], 'files': ['b']}),
-    ],
-)
-def test_asset_rest_path(
-    api_client,
-    draft_version_factory,
-    asset_factory,
-    asset_blob_factory,
-    path,
-    asset_paths,
-    expected,
-):
+def test_asset_rest_path(api_client, draft_version_factory, asset_factory):
```

Comment:
I thought that the idea was to have no effects on API, so am I right to assume that this test should have still worked with all parametrizations? (I would have preferred to keep it even if you think it would not be "as useful as before")

Reply:
> I thought that the idea was to have no effects on API, so am I right to assume that this test should have still worked with all parametrizations? (I would have preferred to keep it even if you think it would not be "as useful as before")

The parametrization was removed here as it was no longer relevant to test this from the asset tests, and similar tests exist in the asset paths tests.